pkg/app/pagewriter/error_page.go

```diff
@@ -37,11 +38,25 @@ type errorPageWriter struct {
 	debug bool
 }
 
+// ErrorPageOpts bundles up all the content needed to write the Error Page
+type ErrorPageOpts struct {
+	// HTTP status code
+	Status int
+	// Redirect URL for "Go back" and "Sign in" buttons
+	RedirectURL string
+	// The UUID of the request
+	RequestID string
+	// App Error shown in debug mode
+	AppError string
+	// Generic error messages shown in non-debug mode
+	Messages []interface{}
```

Comment:
Do you think we should just make this a string and move the formatting out? Did I already ask this? :thinking:

Reply:
I could go either way. When I first started getting these options together in a struct I had the same thought. But the helper functions that wrapped it with `...interface{}` made it mostly a non-factor besides the one ugly test case you mentioned.

Reply:
Let's leave it for now and see if it becomes an issue, as you say, the annoying test case is the primary weird looking bit